server/grpc/server.go

```diff
@@ -19,10 +19,8 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-const tmFlag = "with-tendermint"
-
 // StartGRPCServer starts a gRPC server on the given address.
-func StartGRPCServer(clientCtx client.Context, app types.Application, cfg config.GRPCConfig) (*grpc.Server, error) {
+func StartGRPCServer(clientCtx client.Context, app types.Application, cfg config.GRPCConfig, withTm bool) (*grpc.Server, error) {
```

Comment:
[nit]
`withTm` => `enableCoreGRPC` or similar to convey the purpose of the parameter